src/layouts/Footer.astro

```diff
@@ -1,6 +1,82 @@
 ---
+import { Icon } from 'astro-icon'
 ---
 
 <footer>
-  Footer
+  <style>
+    .nixlogo {
```

Comment:
not sure what's the best way to do this. The good part about this is that it is defined in the same file as it's used. but if we need this svg somewhere else too, we would need a more central place.
Of course, it could also just be an <img> tag that links to an svg file... not sure why it is like this in the original nixos.org implementation?

Reply:
I think we should use the custom icon set as described in astro-icon.